app/connectors/email/connector.py

```
@@ -0,0 +1,79 @@
+"""Email connector: polls email accounts and yields events (stub until full sync)."""
+
+from __future__ import annotations
+
+import asyncio
+import logging
+from collections.abc import AsyncIterator
+from datetime import datetime
+from typing import Any
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)